options.go

```diff
@@ -0,0 +1,52 @@
+package qsc
+
+import "time"
+
+var (
+	_defaultTTL   = 30 * time.Second
+	_defaultDelay = 500 * time.Millisecond
+)
```

Comment:
let's remove these var's and just put them in the options struct we create at the beginning of `New()`